charts/darts-api/values.yaml

```diff
@@ -104,6 +104,8 @@ java:
     DARTS_PORTAL_URL: https://darts.{{ .Values.global.environment }}.apps.hmcts.net
     TESTING_SUPPORT_ENDPOINTS_ENABLED: false
     AUTOMATED_TASK_MODE: false
+    PROCESS_E2E_ARM_RPO: false
+    ARM_RPO_HOURS: 1h
```

Comment:
This is a duration, so probably shouldn't have the word `hours` in it

Reply:
Agreed updated on both flux and this PR